index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 const Build = require('./lib/build');
+const factory = require('./lib/factory');
 const Job = require('./lib/job');
 const Pipeline = require('./lib/pipeline');
 const User = require('./lib/user');
 
-module.exports = { Build, Job, Pipeline, User };
+module.exports = {
+    Build,
+    configureBuildModel: factory.configureBuildModel,
```

Comment:
Sorry, prototype is the not the right word I was looking for.  Static is the word I was looking for.

Example:
```js
const Build = require('screwdriver-models');
Build.configure(datastore, executor);

const build = new Build();
```